bitnami/redis/templates/_helpers.tpl

```diff
@@ -353,3 +353,26 @@ but Helm 2.9 and 2.10 does not support it, so we need to implement this if-else
     {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Compile all warnings into a single message, and call fail.
+*/}}
+{{- define "redis.validateValues" -}}
+{{- $messages := list -}}
+{{- $messages := append $messages (include "redis.validateValues.spreadConstraints" .) -}}
+{{- $messages := without $messages "" -}}
+{{- $message := join "\n" $messages -}}
+
+{{- if $message -}}
+{{-   printf "\nVALUES VALIDATION:\n%s" $message | fail -}}
+{{- end -}}
+{{- end -}}
+
+{{/* Validate values of Redis - spreadConstrainsts K8s version */}}
+{{- define "redis.validateValues.spreadConstraints" -}}
+{{- if and (semverCompare "<1.18-0" .Capabilities.KubeVersion.GitVersion) redis.validateValues.spreadConstraints -}}
+redis: spreadConstraints
+    Pod Topology Spread Constraints are only available on K8s  >= 1.8
```

Comment:
I have detected a minor typo here when testing it: 
```suggestion
{{- if and (semverCompare "<1.18-0" .Capabilities.KubeVersion.GitVersion) .Values.slave.spreadConstraints -}}
redis: spreadConstraints
    Pod Topology Spread Constraints are only available on K8s >= 1.8
```

Reply:
And now that I look at that section it does a version check for `<1.18-0` but the feature seems to have been [added in 1.16](https://v1-16.docs.kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/).  Then the message says 1.8.  Should this also be changed to:

`semverCompare "<1.16-0"`
and
`only available on K8s >= 1.16`

Reply:
Thanks for pointing out. Yes, please, change that as well.

Reply:
done and done.